snippets/csharp/System/NullReferenceException/Overview/Chain2.cs

```diff
@@ -1,64 +1,71 @@
 ﻿// <Snippet7>
 using System;
 
-public class Example
+public class Chain2Example
 {
-   public static void Main()
-   {
-      var pages = new Pages();
-      Page current = pages.CurrentPage;
-      if (current != null) {
-         String title = current.Title;
-         Console.WriteLine("Current title: '{0}'", title);
-      }
-      else {
-         Console.WriteLine("There is no page information in the cache.");
-      }
-   }
+    public static void Main()
+    {
+        var pages = new Pages();
+        Page current = pages.CurrentPage;
+        if (current != null)
+        {
+            string title = current.Title;
+            Console.WriteLine("Current title: '{0}'", title);
```

Comment:
string interpolation:

```suggestion
            Console.WriteLine($"Current title: '{title}'");
```